src/commands/service/create.decorator.json

```diff
@@ -77,6 +77,11 @@
       "arg": "--path-prefix-major-version <path-prefix-major-version>",
       "description": "Version to be used in the path prefix; will be used to configure Traefik2 IngressRoutes. ie. 'v1' will result in a path prefix of '/v1/servicename",
       "defaultValue": ""
+    },
+    {
+      "arg": "--access-token-variable <env-variable>",
```

Comment:
Do we want to also combine this with --helm-config-* since it's only used for that configuration?

Reply:
done